packages/zoe/src/zoe.chainmail

```diff
@@ -164,3 +157,11 @@ interface ZoeGoverningContractFacet () {
   getOffer ( offerHandle :Object) -> (List(OfferRecord));
 
 }
+
+/**
+ * `active` and `inactive` lists of offerHandles.
+ */
+ struct OfferStatusesRecord ( ) {
+  active :List(Object);
```

Comment:
Do we have any kind of typedef? (I forget)

Would be nice to do something like

```
active :List(OfferHandle);
```

and likewise throughout this file.